packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart

```diff
@@ -149,6 +149,21 @@ abstract class CameraPlatform extends PlatformInterface {
     throw UnimplementedError('resumeVideoRecording() is not implemented.');
   }
 
+  /// A new streamed frame is available.
+  ///
+  /// Listening to this stream will start streaming, and canceling will stop.
+  /// Pausing will throw a [CameraException], as pausing the stream would cause
+  /// very high memory usage; to temporarily stop receiving frames, cancel, then
+  /// listen again later.
+  ///
+  ///
+  // TODO(bmparr): Add options to control streaming settings (e.g.,
+  // resolution and FPS).
+  Stream<CameraImageData> onStreamedFrameAvailable(int cameraId,
```

Comment:
I'm confused by this name. Does the app-facing package notify the platform implementation package that a frame is available? Based on the `MethodChannel` implementation, it looks like this method creates the stream that the app-facing package listens to. Would a better name be something like `createCameraImageDataStream`?

Reply:
I don't like this name either; I used this name because it's the pattern used by [all of the existing Streams in the interface](https://github.com/flutter/plugins/blob/main/packages/camera/camera_platform_interface/lib/src/platform_interface/camera_platform.dart#L70-L100)

Do you want to pick a new naming scheme for platform interface streams in this plugin as part of this PR? If so I could make alternate named versions of all of the existing ones, doing passthroughs to the old names (which would be soft-deprecated with a comment), so that we'd get consistency.

Reply:
Ah, I see. I think I was confused by the call to [startImageStream](https://github.com/flutter/plugins/pull/5783/files#diff-8d80e3b1252c652ffc58ad176f59eb21263bf94dee032821ea7d61b917a6bca8R295). I understand now that the app-facing package is adding a listener to the stream which is created by the platform implementation. I think the current name is appropriate for how it is being used then. Streams and callbacks in Flutter tend to use the `on<event-name>` pattern like the one here.